File: ASP.net/DTOs/TaskDto.cs
namespace testing_mode.Models.Dto
{
    public class TaskDto
    {
        public int TaskId { get; set; }
        public int ProjectId { get; set; }
        public string TaskName { get; set; }
        public string AssignedEmployeeName { get; set; }
        public DateOnly StartDate { get; set; }  // Changed to DateOnly
        public DateOnly EndDate { get; set; }    // Changed to DateOnly
        public string TaskDetails { get; set; }
        public int PercentageCompleted { get; set; }
        public string Status { get; set; }
    }
}

File: src/app/DTOs/TaskDto.ts
export interface Task {
  status: string;
  taskId: number;
  projectName: string;
  projectId: number;
  taskName: string;
  startTime: Date;
  endTime: Date;
  assignedEmployeeName: string;
  taskDetails: string;
  progressPercentage: number;
}
